src/Compilers/CSharp/Portable/PublicAPI.Shipped.txt

```diff
@@ -870,13 +870,7 @@ Microsoft.CodeAnalysis.CSharp.Syntax.FixedStatementSyntax.WithFixedKeyword(Micro
 Microsoft.CodeAnalysis.CSharp.Syntax.FixedStatementSyntax.WithOpenParenToken(Microsoft.CodeAnalysis.SyntaxToken openParenToken) -> Microsoft.CodeAnalysis.CSharp.Syntax.FixedStatementSyntax
 Microsoft.CodeAnalysis.CSharp.Syntax.FixedStatementSyntax.WithStatement(Microsoft.CodeAnalysis.CSharp.Syntax.StatementSyntax statement) -> Microsoft.CodeAnalysis.CSharp.Syntax.FixedStatementSyntax
 Microsoft.CodeAnalysis.CSharp.Syntax.ForEachStatementSyntax
-Microsoft.CodeAnalysis.CSharp.Syntax.ForEachStatementSyntax.CloseParenToken.get -> Microsoft.CodeAnalysis.SyntaxToken
```

Comment:
These are still present in the source. The tool excludes them from the API list because they override the method from the base.

---
In reply to: [73446079](https://github.com/dotnet/roslyn/pull/12917#discussion_r73446079) [](ancestors = 73446079)